vms/evm/uptimetracker/uptime_tracker.go

```diff
@@ -175,70 +175,57 @@ func (u *UptimeTracker) update(
 	}
 
 	// Add or update validators
-	for validationID, next := range newValidators {
-		if prev, ok := u.state.getValidatorByValidationID(validationID); ok {
-			// We are updating a validator we know about
-			isActiveUpdated := prev.IsActive == next.IsActive
```

Comment:
This bool was very poorly named - it was named the opposite of what it actually did